tasks/agent.py

```diff
@@ -192,7 +186,7 @@ def build(
             out="cmd/agent/rsrc.syso",
         )
     else:
-        bundled_agents += bundle or BUNDLED_AGENTS.get(flavor, [])
+        bundled_agents += bundle
```

Comment:
`bundle` is `None` by default, I think you need to handle that case
```suggestion
        bundled_agents += bundle or []
```

Reply:
ah you are right, the fault of a last moment cleanup, thanks !